lib/rules/block-closing-brace-empty-line-before/README.md

```diff
@@ -60,3 +60,79 @@ a {
 ```css
 a { color: pink; }
 ```
+
+## Optional secondary options
+
+### `except: ["after-closing-brace"]`
+
+When a rule is nested, and `"never"` is specified for empty lines before closing braces, force a new line before the closing brace of the nested block.
+
```

Comment:
First off, I apologize for the delay in doing this review. Life. Thanks for creating your contribution.

The precedent for the use of "except" in secondary options is to **reverse** the primary option when the excepted case is satisfied. For example see https://github.com/stylelint/stylelint/tree/master/lib/rules/at-rule-empty-line-before#except-after-same-name-inside-block-blockless-after-same-name-blockless-blockless-after-blockless-first-nested and https://github.com/stylelint/stylelint/tree/master/lib/rules/rule-empty-line-before

If I read this correctly it gets half-way there: it reverses the rule when it is "never" but it doesn't do anything when it is "always".